model/distro/distro.go

```diff
@@ -726,6 +732,10 @@ func (d *Distro) GetResolvedPlannerSettings(s *evergreen.Settings) (PlannerSetti
 	if resolved.TargetTime == 0 {
 		resolved.TargetTime = time.Duration(config.TargetTimeSeconds) * time.Second
 	}
+	// If enabled, release mode takes precedent over both distro and admin value.
+	if !s.ServiceFlags.ReleaseModeDisabled && s.ReleaseMode.TargetTimeSecondsOverride > 0 {
+		resolved.TargetTime = time.Duration(s.ReleaseMode.TargetTimeSecondsOverride) * time.Second
```

Comment:
I might be confused but wouldn't we want to lower the target time to make tasks spend less time in the queue?

Reply:
Yes -- presumably the override will be set to something smaller than the regular time!